packages/framework/src/client/resource-request.js

```diff
@@ -15,9 +16,20 @@ export const STATUS = {
 };
 
 export default class ResourceRequest {
-	constructor( resourceName, requirement, resourceState, now ) {
-		this.debug = debugFactory( 'fresh-data:request(' + resourceName + ')' );
+	/**
+	 * Creates a new Resource Request object.
+	 * @param {Object} requirement The requirement for this request (e.g. freshness/timeout)
+	 * @param {Object} resourceState The current state of this request
+	 * @param {string} resourceName The name of the resource
+	 * @param {string} operation The name of the operation to be performed
+	 * @param {Object} data Data to be sent for the operation
+	 * @param {Date} now The current time
+	 */
+	constructor( requirement, resourceState, resourceName, operation, data, now ) {
```

Comment:
In general, I've been adding the default value for those functions designed to be called externally (i.e. public), and not for those functions only called internally to that class/file (i.e. private). In general, I agree with the desire to have continuity, but in this case I don't think it will be adding anything except reducing code coverage.